### PR TITLE
Add a sample module that sets DeviceState for common states

### PR DESCRIPTION
**Pull request by spbrogan**

### CODE REVIEW

File: OemPkg/DeviceStatePei/DeviceStatePei.c [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+  This platform module sets the DEVICE_STATE bits prior to display.
+
+  This driver currently implements the following bits in a standard method:
+    DEVICE_STATE_SECUREBOOT_OFF
+
+  This library can either mask those bits (force low) or add other bits.  See the
+  MdeModulePkg/Include/Library/DeviceStateLib.h file for bit definitions.
+
+  Copyright (C) Microsoft Corporation. All rights reserved.
```
Comment (kuqin12) [inline OemPkg/DeviceStatePei/DeviceStatePei.c]: Copyrights

File: OemPkg/DeviceStatePei/DeviceStatePei.inf [hunk 2]
```diff
@@ -0,0 +1,44 @@
+## @file DeviceStatePei.inf
+#
+#  This module checks a device state platform conditions and sets the state
+#  accordingly.
+#
+#  This driver currently implements the following bits in a standard method:
+#    DEVICE_STATE_SECUREBOOT_OFF
+#
+#
+#  Copyright (C) Microsoft Corporation. All rights reserved.
```
Comment (kuqin12) [inline OemPkg/DeviceStatePei/DeviceStatePei.inf]: Ditto

File: OemPkg/OemPkg.dsc [hunk 3]
```diff
@@ -132,6 +139,9 @@
   OemPkg/Library/DfciDeviceIdSupportLib/DfciDeviceIdSupportLib.inf
   OemPkg/FrontpageButtonsVolumeUp/FrontpageButtonsVolumeUp.inf
 
+  [Components.IA32]
```
Comment (kuqin12) [inline OemPkg/OemPkg.dsc]: nit: alignment.

**Review by kuqin12 — APPROVED** [review]

**Comment by matthewfcarlson** [comment]
Description would be nice :)

**Review by matthewfcarlson — APPROVED** [review]

**Pull request merged**